src/hooks/useFeaturedProducts.js

```diff
@@ -1,12 +1,14 @@
-import useData from 'hooks/useData';
+import { useAPIData } from 'utils/hooks/useAPIData';
 
 const useFeaturedProducts = () => {
-    const fileName = 'featured-products.json';
+    const queries = ['at(document.type, "product")', 
```

Comment:
I don't know how difficult it is, but something cool i think you can do is merging `useFeaturedBanners` and `useFeaturedProducts`

Reply:
I know it may not be easy because the map is not the same, an also de queries are different but I think is possiblel

Reply:
You mean merging both hooks so it returns [productCategories, featuredBanners], right? It can be done. I could even implement useMemo to improve performance in case one of the API calls is not needed.